java-manta-it/src/test/java/com/joyent/manta/client/MantaClientSigningIT.java

```diff
@@ -261,7 +263,7 @@ public final void testCanCreateSignedOPTIONSUriFromPath()
                 Assert.fail(errorText);
             }
 
-            String server = connection.getHeaderField("Server");
+            String server = StringUtils.substringBefore(connection.getHeaderField("Server"), SEPARATOR);
```

Comment:
Tho this server header uses `/` and `com.joyent.manta.client.MantaClient.SEPARATOR` happens to also be `/`, I don't think this code needs to related them. I'd be inclined to use a literal `'/'` here. Does that work?

Reply:
Feedback accounted for: refactored `SEPARATOR -> "/" literal`